lessons/day1/ruby_types.md

```diff
@@ -21,11 +21,11 @@ To determine the type of an object you can use `.class`
 
 ```ruby
 x = 1
-puts x.class #Fixnum
+puts x.class #Integer
 
 word = "hello"
 puts word.class #String
 ```
 
 ### Resources
-* [Ada Ruby Types Video](https://adaacademy.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=0a3aad23-0e7d-4dca-b6bb-f1ed3bed8692) (5:52)
\ No newline at end of file
+* [Ada Ruby Types Video](https://adaacademy.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=0a3aad23-0e7d-4dca-b6bb-f1ed3bed8692) (5:52)
```

Comment:
Not sure why this line shows as being edited - I don't see any difference.

I did make this change via the github UI, maybe that added/removed a tailing newline?